python/ray/_raylet.pyx

```diff
@@ -4621,6 +4621,10 @@ cdef class CoreWorker:
         return (CCoreWorkerProcess.GetCoreWorker().GetWorkerContext()
                 .CurrentActorIsAsync())
 
+    def exit_current_actor(self):
+        return (CCoreWorkerProcess.GetCoreWorker().GetWorkerContext()
+                .SetCurrentActorShouldExit())
```

Comment:
```suggestion
    def set_current_actor_should_exit(self):
        return (CCoreWorkerProcess.GetCoreWorker().GetWorkerContext()
                .SetCurrentActorShouldExit())
```

No reason to have mismatching names (which is confusing to read), and this name better conveys the behavior. As written, `exit_current_actor` implies that the actor would exit immediately.